js/lib/beautify.js

```diff
@@ -1673,13 +1674,21 @@
                 var allow_e = true;
                 var local_digit = digit;
 
-                if (c === '0' && parser_pos < input_length && /[Xxo]/.test(input.charAt(parser_pos))) {
-                    // switch to hex/oct number, no decimal or e, just hex/oct digits
+                if (c === '0' && parser_pos < input_length && /[Xxob]/.test(input.charAt(parser_pos))) {
+                    // switch to hex/oct/bin number, no decimal or e, just hex/oct/bin digits
                     allow_decimal = false;
                     allow_e = false;
                     c += input.charAt(parser_pos);
                     parser_pos += 1;
-                    local_digit = /[o]/.test(input.charAt(parser_pos)) ? digit_oct : digit_hex;
+                    if ( /[b]/.test(input.charAt(parser_pos)) ) {
+                        local_digit = digit_bin;
+                    } else {
```

Comment:
I disagree. That is not officially valid Javascript. There is no "else if" in the language. The "else" is always put in a else block. While the language will handle "else if", I am being explicit for clarity sake and to allow to extension of the else block.

This seems to be a nitpick for accepting a pull request.